GANDLF/losses/regression.py

```diff
@@ -74,7 +76,12 @@ def CE(prediction, target):
     return loss_val
 
 
-def CCE_Generic(prediction, target, params, CCE_Type):
+def CCE_Generic(
+    prediction: torch.Tensor,
+    target: torch.Tensor,
+    params: dict,
+    CCE_Type: torch.nn = CrossEntropyLoss(),
```

Comment:
Should the class be initialized here, or just passed as an object? I mean from CrossEntropyLoss() change to CrossEntropyLoss

Reply:
Fixed.